contracts/src/oracle/library.cairo

```diff
@@ -108,6 +108,10 @@ func UpdatedCurrency(currency: Currency) {
 func SubmittedPair(pair: Pair) {
 }
 
+@event
+func CheckpointSpotEntry(pair_id: felt) {
```

Comment:
might be worth emitting the checkpointed data too? (`price`, `decimals`, `last_updated_timestamp`, `num_sources_aggregated`)